source/_components/climate.knx.markdown

```diff
@@ -112,6 +135,24 @@ operation_mode_comfort_address:
   description: KNX address for switching on/off comfort mode.
   required: false
   type: string
-{% endconfiguration %}
-
-`operation_mode_frost_protection_address` / `operation_mode_night_address` / `operation_mode_comfort_address` are not necessary if `operation_mode_address` is specified.
+operation_modes:
+  description: Overrides the supported operation modes.
+  required: false
```

Comment:
What happens if the user doesn't specify this? What operation modes will be activated?